vendor/src/github.com/docker/libnetwork/network.go

```diff
@@ -60,14 +60,23 @@ type network struct {
 }
 
 func (n *network) Name() string {
+	n.Lock()
+	defer n.Unlock()
+
 	return n.name
```

Comment:
@tiborvass In this particular case I don't necessarily agree that we should avoid defer. We dont need extra boiler plate of introducing a temporary variable just for this. defer works nicely here and helps readabiity.